components/Tabs/examples/TabBarIconExample.swift

```diff
@@ -0,0 +1,122 @@
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialPalettes
+import MaterialComponents.MaterialTabs
+
+class TabBarIconSwiftExample: UIViewController {
+
+  // MARK: Properties
+  var alignment: MDCTabBarAlignment {
+    get {
+      return tabBar.alignment
+    }
+    set(newAlignment) {
+      tabBar.setAlignment(newAlignment, animated: true)
+    }
+  }
+
+  lazy var alignmentButton: MDCRaisedButton = self.setupAlignmentButton()
+
+  lazy var appBar: MDCAppBar = self.setupAppBar()
+
+  lazy var scrollView: UIScrollView = self.setupScrollView()
+
+  lazy var starPage: UIView = self.setupStarPage()
+
+  lazy var tabBar: MDCTabBar = {
+    let tabBar = MDCTabBar()
+    tabBar.delegate = self
+    tabBar.alignment = .centerSelected
+
+    let bundle = Bundle(for: type(of: self))
+    let info = UIImage.init(named: "TabBarDemo_ic_info", in: bundle, compatibleWith:nil)
+    let star = UIImage.init(named: "TabBarDemo_ic_star", in: bundle, compatibleWith:nil)
+
+    tabBar.items = [UITabBarItem(title: "Info", image: info, tag:0),
+                    UITabBarItem(title: "Stars", image: star, tag:0)]
+    tabBar.items[1].badgeValue = "1"
+
+    let blue = MDCPalette.blue().tint500
+    tabBar.tintColor = blue
+    tabBar.inkColor = blue
+
+    tabBar.barTintColor = UIColor.white
+    tabBar.itemAppearance = .titledImages
+    tabBar.selectedItemTintColor = UIColor.black.withAlphaComponent(0.87)
+    tabBar.unselectedItemTintColor = UIColor.black.withAlphaComponent(0.38)
+
+    return tabBar
+  }()
+
+  // MARK: Methods
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    setupExampleViews()
+
+    alignmentButton.addTarget(self, action:#selector(changeAlignmentDidTouch(sender:)), for: .touchUpInside)
+  }
+
+  func changeAlignmentDidTouch(sender: UIButton) {
+    let sheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+    sheet.addAction(UIAlertAction(title: "Leading", style: .default, handler: { _ in
+      self.alignment = .leading
+    }))
+    sheet.addAction(UIAlertAction(title: "Center", style: .default, handler: { _ in
+      self.alignment = .center
+    }))
+    sheet.addAction(UIAlertAction(title: "Justified", style: .default, handler: { _ in
+      self.alignment = .justified
+    }))
+    sheet.addAction(UIAlertAction(title: "Selected Center", style: .default, handler: { _ in
+      self.alignment = .centerSelected
+    }))
+    present(sheet, animated: true, completion:nil)
+  }
+
+  func incrementStarBadge() {
+    let starItem = tabBar.items[1]
+    guard let badgeValue = starItem.badgeValue,
+      let badgeNumber = Int(badgeValue), badgeNumber > 0 else {
+      return
+    }
+
+    starItem.badgeValue = NumberFormatter.localizedString(from:(badgeNumber + 1) as NSNumber,
```

Comment:
How is the alignment supposed to go here? Should the starts of 'from' and 'number' be aligned?